packages/shared-base/src/Messages/Mask.ts

```diff
@@ -10,6 +10,69 @@ enum NetworkPluginID {
     PLUGIN_SOLANA = 'com.mask.solana',
 }
 
+// Learn more about ethereum ChainId https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
+export enum ChainId {
```

Comment:
Please avoid doing this. Just use `number`.

Reply:
why? 🤔